base_layer/wallet/tests/output_manager_service/service.rs

```diff
@@ -170,8 +172,8 @@ fn sending_transaction_and_confirmation<T: Clone + OutputManagerBackend + 'stati
         MicroTari::from(100 + OsRng.next_u64() % 1000),
         &factories.commitment,
     );
-    runtime.block_on(oms.add_output(uo.clone())).unwrap();
-    match runtime.block_on(oms.add_output(uo)) {
+    runtime.block_on(oms.add_output(uo.clone().into())).unwrap();
```

Comment:
all of these .into() instances seem unnecessary. I don't think you should leak the use of DbUnblindedOutput out of the OMS which you do by changing the make_input(...) function in the wallet utils.